mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -30,6 +30,7 @@ CLASS net/minecraft/class_1321 net/minecraft/client/MinecraftClient
 	FIELD field_26842 musicTracker Lnet/minecraft/class_6104;
 	FIELD field_26844 sessionService Lcom/mojang/authlib/minecraft/MinecraftSessionService;
 	FIELD field_26845 skinProvider Lnet/minecraft/class_6048;
+	FIELD field_26847 clientThread Ljava/lang/Thread;
```

Comment:
`thread`, we talked about this on discord.

Reply:
![image](https://user-images.githubusercontent.com/25501603/105951991-9e3f5180-60bc-11eb-93cb-feb34386ac94.png)